packages/fbtee/package.json

```diff
@@ -21,12 +21,14 @@
   "type": "module",
   "main": "lib/index.js",
   "scripts": {
-    "build": "tsup src/index.tsx -d lib --target=node22 --format=esm --clean --no-splitting --dts"
+    "build": "pnpm build:babel && pnpm build:prepend && pnpm build:fbtee-strings && tsup lib-tmp/index.tsx -d lib --target=node22 --format=esm --clean --no-splitting --dts",
+    "build:babel": "babel --delete-dir-on-start --copy-files --config-file ./babel-build.config.js --out-dir=lib-tmp --extensions=.tsx --keep-file-extension --ignore='src/**/__tests__/*.tsx' src",
+    "build:fbtee-strings": "pnpm fbtee manifest && pnpm fbtee collect --pretty --include-default-strings=false --manifest < .src_manifest.json > Strings.json $(find src -type f \\! -path '*/__tests__/*' \\! -path '*/__mocks__/*') && ./scripts/rewrite-filepaths.ts",
```

Comment:
This is what generates the default strings that will be included in downstream projects.